### DISCUSSION TITLE
《Loopcraft2020-2021》

### DISCUSSION THREAD

**Discussion by Lupingyue** [qihuju/qihuju.github.io#35] [Show and tell]
[《Loopcraft2020-2021》NFT计划](https://github.com/qihuju/qihuju.github.io/projects/3)

- Loopcraft脑图
「点击查看“岳路平脑图个展”：https://qihuju.wiki/docs/qihuju-links/yueluping.html 」

- Loopcraft server计划 with @dadabit 「思考中💭」

**Comment by dadabit** [comment]
思考中

**Reply by Lupingyue** [reply]
> 思考中

有什么spark，随时在这里讨论😄

**Reply by dadabit** [reply]
seven park

**Reply by Lupingyue** [reply]
seven park是什么？科普一下

**Reply by dadabit** [reply]
我也不知道

**Reply by dadabit** [reply]
Brainstorm

**Reply by Lupingyue** [reply]
> Brainstorm

嗯嗯😄

**Reply by yueyang2024037683** [reply]
怎么进入您开发的这个minecraft服务器呢

**Comment by Lupingyue** [comment]
你是？你有微信吗？



发自我的iPhone


------------------ Original ------------------
From: 악양 ***@***.***&gt;
Date: Fri,Sep 18,2026 10:01 PM
To: qihuju/qihuju.github.io ***@***.***&gt;
Cc: Lupingyue ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [qihuju/qihuju.github.io] 《Loopcraft2020-2021》 (Discussion #35)





 
怎么进入您开发的这个minecraft服务器呢
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS and Android. Download it today!
